src/model/wallet.ts

```diff
@@ -257,11 +258,17 @@ export const walletInit = async (
   return { isNew, walletAddress };
 };
 
-export const loadWallet = async (
-  address?: EthereumAddress | undefined,
-  showErrorIfNotLoaded = true,
-  provider?: Provider
-): Promise<null | Wallet | LedgerSigner> => {
+export const loadWallet = async <S extends Screen>({
```

Comment:
The following is really just converting the `loadWallet` method to object parameters so that we can pass more than one optional.